analysis/benchmark_results.py

```diff
@@ -247,6 +247,18 @@ def coverage_growth_plot(self):
             self._benchmark_df, self._get_full_path(plot_filename), wide=True)
         return plot_filename
 
+    @property
+    def coverage_growth_plot_logscale(self):
+        """Coverage growth plot."""
+        plot_filename = self._prefix_with_benchmark(
```

Comment:
since coverage_growth_plot_logscale and coverage_growth_plot share all the code except logscale arg, can you create a common helper and use in both places.

Reply:
just fixed this as well

Reply:
Here's an example chart:

![image](https://user-images.githubusercontent.com/558033/98165085-488b4c80-1eb3-11eb-9616-200c4b1c981e.png)

Reply:
I can't upload the zip  for some reason, but here's the [example report](https://tr0gd0r.com/download/example_dual_2020-11-01).